src/Joonasw.AspNetCore.SecurityHeaders/Csp/CspNonceService.cs

```diff
@@ -6,6 +6,12 @@ namespace Joonasw.AspNetCore.SecurityHeaders.Csp
     public class CspNonceService : ICspNonceService
     {
         private readonly string _nonce;
+        private static char base64PadCharacter = '=';
+        private static string doubleBase64PadCharacter = "==";
+        private static char base64Character62 = '+';
```

Comment:
IMO these should be `const`, also the naming style should then be PascalCase

Reply:
made the requested changes, sorry about the delay.